openwisp_users/models.py

```diff
@@ -39,6 +39,7 @@ class User(AbstractUser):
     OpenWISP User model
     """
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    last_name = models.CharField(_('last name'), max_length=30, blank=True)
```

Comment:
@nemesisdesign The max_length attribute of last_name has been changed to 150 in django >2.0. But it was 30 before. So travis breaks thinking a migration is needed.

Reply:
Actually replied here long back but I noticed that it was in pending review where I didn't submit it. One of the github confusions.

Reply:
@rohithasrk `last_name` to 150? That's surprising.. but I'd like to avoid having to override the default fields of the user model because that way we can take advantage of any improvements made in django that we are not aware.
We should override the default fields only when really necessary and this case doesn't look that important.
Can we simply create a new migration that sets the `last_name` to the value used by django?